android/app/src/main/res/layout/item_bookmark_category.xml

```diff
@@ -5,15 +5,17 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
+    android:paddingStart="@dimen/margin_quarter"
+    android:paddingEnd="@dimen/dp_0"
   android:background="?clickableBackground"
   android:gravity="center_vertical">
-  <CheckBox
+  <com.google.android.material.checkbox.MaterialCheckBox
```

Comment:
this is the thing i'm unsure about, it's extended from androidx appcompatcheckbox, but I'm not sure how to check if it's good for api 21.

the reason for using is that with a standard checkbox, it's very hard to centre the icon 😅 this one just works though

https://developer.android.com/reference/com/google/android/material/checkbox/MaterialCheckBox

Reply:
Are there other solutions without using Compat or Material? E.g. several solutions listed here: https://stackoverflow.com/a/3965796

Reply:
Use material checkbox has no impact if you continue to use compat themes.